api.js

```diff
@@ -0,0 +1,164 @@
+"use strict";
+
+const {classes: Cc, interfaces: Ci, results: Cr, utils: Cu} = Components;
+
+Cu.import("resource://gre/modules/XPCOMUtils.jsm");
+
+Cu.import("resource://devtools/shared/event-emitter.js");
+Cu.import("resource://gre/modules/ExtensionUtils.jsm");
+const {SingletonEventManager} = ExtensionUtils;
+
+XPCOMUtils.defineLazyModuleGetter(this, "webrtcUI",
+                                  "resource:///modules/webrtcUI.jsm");
+
+let origReceiveMessage = null;
```

Comment:
Agreed, I had a separate conversation with @jan-ivar about making the fix in webrtcUI.jsm.  He's on board with fixing it there and that's in my queue of things to do.  That change will need to land in central and ride the trains though, I'll aim to get it done before 52 is cut.